src/components/slideshow/slideshow.js

```diff
@@ -201,14 +201,16 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
         function onAutoplayStart() {
             var btnSlideshowPause = dlg.querySelector('.btnSlideshowPause i');
             if (btnSlideshowPause) {
-                btnSlideshowPause.innerHTML = "pause";
+                btnSlideshowPause.classList.remove("play_arrow");
+                btnSlideshowPause.classList.add("pause");
```

Comment:
```suggestion
                btnSlideshowPause.classList.replace("play_arrow", "pause");
```

Reply:
have to be polyfilled

Reply:
We could use [DOM4](https://www.npmjs.com/package/dom4) to polyfill it, as well as a bunch of other things.